src/loaders/express/tests/helmet.spec.ts

```diff
@@ -0,0 +1,137 @@
+import helmet from 'helmet'
+import { mocked } from 'ts-jest/utils'
+
+import config from 'src/config/config'
+import featureManager from 'src/config/feature-manager'
+
+import helmetMiddlewares from '../helmet'
+
+describe('helmetMiddlewares', () => {
+  jest.mock('helmet')
+  const mockHelmet = mocked(helmet, true)
+  jest.mock('src/config/feature-manager')
+  const mockFeatureManager = mocked(featureManager, true)
+  jest.mock('src/config/config')
+  const mockConfig = mocked(config, true)
+
+  beforeEach(() => {
+    mockHelmet.xssFilter = jest.fn().mockImplementation(() => 'xssFilter')
+    mockHelmet.noSniff = jest.fn().mockImplementation(() => 'noSniff')
+    mockHelmet.ieNoOpen = jest.fn().mockImplementation(() => 'ieNoOpen')
+    mockHelmet.dnsPrefetchControl = jest
+      .fn()
+      .mockImplementation(() => 'dnsPrefetchControl')
+    mockHelmet.hidePoweredBy = jest
+      .fn()
+      .mockImplementation(() => 'hidePoweredBy')
+    mockHelmet.referrerPolicy = jest
+      .fn()
+      .mockImplementation(() => 'referrerPolicy')
+    mockHelmet.contentSecurityPolicy = jest
+      .fn()
+      .mockImplementation(() => 'contentSecurityPolicy')
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+
+  it('should call the correct helmet functions', () => {
+    helmetMiddlewares()
+    expect(mockHelmet.xssFilter).toHaveBeenCalled()
+    expect(mockHelmet.noSniff).toHaveBeenCalled()
+    expect(mockHelmet.ieNoOpen).toHaveBeenCalled()
+    expect(mockHelmet.dnsPrefetchControl).toHaveBeenCalled()
+    expect(mockHelmet.hidePoweredBy).toHaveBeenCalled()
+    expect(mockHelmet.referrerPolicy).toHaveBeenCalled()
+    expect(mockHelmet.contentSecurityPolicy).toHaveBeenCalled()
```

Comment:
should expect `helmet.hsts` to have been called with certain arguments as well right?

Reply:
or rather we need to ensure that `hstsMiddleware` behaves correctly. perhaps it should have its own separate unit test?

Reply:
Added in the tests - see if ok